Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/configadmin/ConfigAdmin.java

```diff
@@ -599,18 +613,17 @@ private X509Certificate generateCertificate(KeyPair keypair, String dn, int days
     @Override
     public SimpleCertificateResponseMessageType deleteTemporaryKeystore(
         DeleteTemporaryKeystoreRequestMessageType request) {
-        File deleteGateway = new File(FILE_JKS_GATEWAY_NEW);
-        File deleteCaCert = new File(FILE_JKS_CACERTS_NEW);
-        tempKeystore = null;
-        tempTruststore = null;
+        File tmpFolder = new File(DIR_NHINC_PROPERTIES_TEMP);
 
         boolean deleted = false;
-
-        if (deleteGateway.exists() && deleteGateway.delete()) {
-            deleted = true;
-        }
-        if (deleteCaCert.exists() && deleteCaCert.delete()) {
-            deleted = true;
+        if (tmpFolder.exists()) {
+            try {
+                FileUtils.deleteDirectory(tmpFolder);
+                deleted = true;
+            } catch (IOException e) {
+                LOG.error("Error while cleaning up the tmp folder: {}", e.getLocalizedMessage(), e);
+                return buildSimpleResponse(false, "Error whild cleaning up the temporary folder.");
```

Comment:
whild -> "while"

Can't comment directly due to unchanged line - 
"Temporary files are deleted: successful." : "Temporary files does not exist." ->
"Temporary files were deleted successfully." : "Temporary files did not exist. Skipping temporary file cleanup"

The latter message makes it sound as if this is an error.